faq.html

```diff
@@ -155,10 +155,13 @@ <h2 id="general">General</h2>
       <a href="{{page.root}}/img/software-carpentry-banner.png">Software Carpentry logo</a>
       are both trademarked,
       and may only be used with our explicit prior permission.
-      We normally grant this to any training course that covers our
-      <a href="#core-topics">core topics</a>
-      has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,
-      and sends us summary information about attendees.
+      We normally grant this to any training course that:
+      <ul> 
+       <li>covers our <a href="#core-topics">core topics</a>, </li>
+       <li>has at least one <a href="{{page.root}}/pages/team.html">certified instructor</a> teaching,</li>
+       <li>runs our standardised pre- and post-workshop questionnaires and provides us with the results,</li>
```

Comment:
So, you're saying it is acutally impossible to have a self-organized workshop because it's impossible to "run our standardized questionsairres and provide the results" ?
This is a serious circular issue.